lib/records/RecCore.cc

```diff
@@ -514,6 +514,18 @@ RecLookupMatchingRecords(unsigned rec_type, const char *match, void (*callback)(
   return REC_ERR_OKAY;
 }
 
+void
+RecLookupIterateRecords(std::function<void (RecRecord const*)> f)
```

Comment:
Yes, I saw the `RecLookup...` functions and modeled the new call on those. I didn't see `RecDumpRecords`.